app/(shop)/[...category]/page.tsx

```diff
@@ -0,0 +1,232 @@
+                </div>
+                {/* Categories List */}
+                <div className={classNames('flex flex-wrap mx-auto gap-2  max-w-(--breakpoint-2xl) empty:pb-0 pb-4 ')}>
+                    {categories?.map((child) => {
+                        return (
+                            <Link
+                                className={classNames(
+                                    'group w-28 pt-2 text-center text-dark divide divide-black divide-solid hover:border-dark transition-all',
+                                    'bg-light border-muted border border-solid rounded-lg  flex flex-col gap-1  justify-start  items-center',
+                                )}
+                                /*@ts-expect-error*/
+                                href={child?.path}
+                                /*@ts-expect-error*/
+                                key={child?.id}
+                            >
+                                <div className="w-24 h-24 text-center rounded-lg overflow-hidden border border-muted relative ">
+                                    {/*@ts-expect-error*/}
+                                    {child.image?.map((img) => {
+                                        /*@ts-expect-error*/
+                                        return <Image {...img} key={`${img?.url}`} alt={child?.name} sizes="200px" />;
+                                    })}
+                                </div>
+                                <span className="group-hover:font-bold py-2 text-sm text-wrap max-w-full">
+                                    {/*@ts-expect-error*/}
+                                    {child?.name}
+                                </span>
+                            </Link>
+                        );
+                    })}
+                </div>
+            </div>
+
+            {/* Blocks */}
+            {blocks && (
+                <div className={classNames('flex flex-col items-center pt-12', !!blocks?.length && 'pb-12')}>
+                    <Blocks blocks={blocks} />
+                </div>
+            )}
+
+            {/* Products List */}
+            <div
+                className={classNames(
+                    'grid  mt-2 grid-cols-1 gap-6 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 lg:gap-4 max-w-(--breakpoint-2xl) mx-auto mb-8 relative',
+                )}
+            >
+                <div className="col-span-1 sm:col-span-2 md:col-span-3 lg:col-span-4 pb-4 mt-4">
+                    {/* Filters */}
+                    <Suspense fallback={null}>
+                        <Filters
+                            priceRange={priceRangeOptions}
+                            sorting={sort}
+                            totalHits={totalHits ?? 0}
+                            paths={paths}
+                            stockOptions={stockOptions}
+                        />
+                    </Suspense>
+                </div>
+                {/*@ts-expect-error*/}
```

Comment:
One more... lets get rid of all of them as this is public code and it does not look good